### PR TITLE
Exporting our camp target except if another one is provided

### PR DESCRIPTION
**Pull request by adrienbernede**
Exporting our camp target expect if another one is provided

### CODE REVIEW

**Comment by adrienbernede** [comment]
LGTM

**Comment by adrienbernede** [comment]
LGTM

File: umpire-config.cmake.in [hunk 1]
```diff
@@ -7,6 +7,11 @@
 get_filename_component(UMPIRE_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 set(UMPIRE_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
 
+if (NOT DEFINED camp_DIR)
```
Comment (davidbeckingsale) [inline umpire-config.cmake.in:10]: I think this whole thing should be inside a `if (NOT TARGET camp)` block.

Comment (adrienbernede) [reply umpire-config.cmake.in]: Good catch!

**Comment by davidbeckingsale** [comment]
LGTM

**Comment by adrienbernede** [comment]
LGTM

**Comment by davidbeckingsale** [comment]
> LGTM

I beat you to it 😂

**Comment by adrienbernede** [comment]
> I beat you to it 😂

You forgot to trigger to the mirroring to give it a boost :p

**Pull request merged**